composer.json

```diff
@@ -17,7 +17,7 @@
         }
     ],
     "require": {
-        "php": "^5.3 || ^7.0",
+        "php": "^7.0",
```

Comment:
Composer is still running with the 7.0 version. A change on this file can only be done with the help of @Soullivaneuh